src/PageCache.php

```diff
@@ -133,7 +133,9 @@ public function init()
             .'; script:'.$_SERVER['SCRIPT_NAME'].'; query:'.$_SERVER['QUERY_STRING'].'.');
 
         // Search for valid cache item for current request
-        if ($item = $this->getCurrentItem()) {
+        $item = $this->getCurrentItem();
```

Comment:
why not keep as was?

Reply:
Assignment inside of the `if` operator considered risky coz it may lead to unexpected results. Also It is not clear for another developer coz it may be understood as an equality check if he/she would be in a rush while reading this code. It is much better to keep assignments separate from `if` and `while` operators (better readability and maintainability).